### PR TITLE
Add more docs.

### PR DESCRIPTION
**Pull request by markstory**

### CODE REVIEW

File: docs/Checking-Authorization.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+application and added an `identity` to the request, you can start checking
+authorization. The middleware will wrap your request `identity` with an
+`IdentityDecorator` that adds authorization related methods:
+
+```php
+// Get the identity from the request
+$user = $this->request->getAttribute('identity');
+
+// Check authorization on $article
+if ($user->can($article, 'delete')) {
```
Comment (robertpustulka) [inline docs/Checking-Authorization.md]: Wrong order of arguments, should be: `$user->can('delete', $article)`

File: docs/Checking-Authorization.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+```
+
+You can also use the `identity` to apply scopes:
+
+```php
+// Get the identity from the request
+$user = $this->request->getAttribute('identity');
+
+// Apply permission conditions to a query
+$query = $user->applyScope($query, 'index');
```
Comment (robertpustulka) [inline docs/Checking-Authorization.md]: As above. Should be: `$user->applyScope('index', $query)`

File: docs/Middleware-and-Component.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+{
+    parent::initialize();
+    $this->loadComponent('Authorization.Authorization');
+}
+```
+
+### Automatic authorization checks
+
+By default `AuthorizationComponent` will attempt to automatically apply
+authorization based on the controller's default table class and current action
```
Comment (robertpustulka) [inline docs/Middleware-and-Component.md]: This applies to *models* in general, not only tables. Should we mention it or leave tables to simplify things?

File: docs/Middleware-and-Component.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+```php
+$this->loadComponent('Authorization.Authorization', [
+    'actionMap' => [
+        'update' => false,
+        'delete' => false,
+    ]
+];
+```
+
+Any action not set to `false` in the `actionMap` will have authorization checked
```
Comment (robertpustulka) [inline docs/Middleware-and-Component.md]: You can mention that you can map controller actions to authorization actions, ie `'update' => 'modify'`

File: readme.md [hunk 5]
```diff
@@ -6,14 +6,37 @@
...
 
-**This is work in progress! Don't use it in a live app!**
+**:warning: This is still under active development and APIs may change 
+  without notice :warning:**
 
-## Authorization is not Authentication
+## Authorization not Authentication
 
-This plugin intentionally **does not** handle authentication. It was [decoupled](https://en.wikipedia.org/wiki/Coupling_(computer_programming)) from authentication on purpose for a clear [separation of concerns](https://en.wikipedia.org/wiki/Separation_of_concerns). See also [Computer access control](https://en.wikipedia.org/wiki/Computer_access_control). This plugin handles only *authorization*. For *authentication* use the [CakePHP Authentication plugin](https://github.com/cakephp/authentication).
+This plugin is intends to provide a framework around authorization and access
```
Comment (robertpustulka) [inline readme.md]: > This plugin is intends

superfluous "is"?

File: docs/Middleware-and-Component.md (hunk 3 above)
Comment (markstory) [reply docs/Middleware-and-Component.md]: I'll update it to say model.

**Comment by markstory** [comment]
@robertpustulka I added more docs for resolvers in general.

**Pull request merged**